test/e2e/common/expansion.go

```diff
@@ -144,4 +149,188 @@ var _ = framework.KubeDescribe("Variable Expansion", func() {
 			"test-value",
 		})
 	})
+
+	/*
+		    Testname: var-expansion-subpath
+		    Description: Make sure a container's subpath can be set using an
+			expansion of environment variables.
+	*/
+	It("should allow substituting values in a volume subpath [Feature:VolumeSubpathEnvExpansion]", func() {
```

Comment:
If you want this run in the alpha node e2e suite, you can add a `[NodeAlphaFeature:VolumeSubpathEnvExpansion]`.

Reply:
Okay Thanks. I'll add that next week and hopefully this will then go green

Reply:
@yujuhong @msau42 Please can you confirm whether I replace the current Feature: with this new one or add it as an additional and have both? And are the test infra tickets now redundant or still applicable?